plugins/synapticsmst/synapticsmst-device.c

```diff
@@ -390,6 +395,11 @@ synapticsmst_device_enumerate_device (SynapticsMSTDevice *device,
 	priv->chip_id = (byte[0] << 8) | (byte[1]);
 	priv->chip_id_str = g_strdup_printf ("VMM%02x%02x", byte[0], byte[1]);
 
+	if (priv->chip_id > 0x5000)
+		priv->chip_kind = SYNAPTICSMST_CHIP_KIND_PANAMERA;
```

Comment:
Could you inquire to Synaptics the board ID's that actually match `panamera`?  I do like the idea of future proofing this code from trying to run on a new board that _might not be compatible_.

It might be as simple as if
```
if (priv->chip_id < 0x5000)
//tesla or leaf
else if (priv->chip_id & 0x5000)
//panamera
else
//unknown